index.html

```diff
@@ -47,115 +57,120 @@ <h3>I’am a Software Developer</h3>
-        <ul class="project-item-links">
-          
-          <li><a href="">html</a></li>
-          <li><a href="">bootstrap</a></li>
-          <li><a href="">Ruby</a></li>
-        </ul>
-        <button class="project-item-btn" type="submit">See Project</button>
+        <div class="project-item">
+          <div class="item-content">
+            <h2>Data Dashboard Healthcare</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="project-item-links">
+              
+              <li><a href="">html</a></li>
+              <li><a href="">bootstrap</a></li>
+              <li><a href="">Ruby</a></li>
+            </ul>
+          </div>
       </div>
+      
       <div class="project-item">
-        <h2>Profesional Art Printing Data</h2>
-        <p>
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard
-        </p>
-        <ul class="project-item-links">
-          <li><a href="">html</a></li>
-          <li><a href="">bootstrap</a></li>
-          <li><a href="">Ruby</a></li>
-        </ul>
-        <button class="project-item-btn" type="submit">See Project</button>
+          <div class="item-content">
+            <h2>Website Portfolio</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="project-item-links">
+              <li><a href="">html</a></li>
+              <li><a href="">bootstrap</a></li>
+              <li><a href="">Ruby</a></li>
+            </ul>
+          </div>
       </div>
     </section>
 
     <section class="aboutme-section">
-      <h2 class="aboutme-title">About me</h2>
-      <p>
-        Hello I’m a software developer! I can help you build a product , feature
-        or website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hestiate to contact me.
-      </p>
-      <button class="aboutme-resumebtn" type="submit">Get My Resume</button>
-      <img class="divider" src="./images/Devider.png" alt="" />
+
+      <div class="about-intro">
+        <h2 class="aboutme-title">About me</h2>
+        <p>
+            Hello I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <button class="aboutme-resumebtn" type="submit">Get My Resume</button>
+        <img class="divider" src="./images/Devider.png" alt="" />
+      </div>
       <div class="aboutme-technologies">
         <ul class="technologies-category">
           <li class="first">
```

Comment:
- [ ] Hey, great job with your design, however, your design is not yet close to the Figma design. Please consider this little adjustment to make it even better..

**YOUR DESIGN**

![now](https://user-images.githubusercontent.com/59397159/167253713-d453c849-1803-4bf2-ad88-8eb054016d8a.png)


**FIGMA DESIGN**

![noee](https://user-images.githubusercontent.com/59397159/167253722-f3863d02-b38a-44c7-9999-997f25001f44.png)

**WHAT TO IMPROVE**
- Image should be aligned to the left
- Increase the space from top and bottom for the horizontal line
- The `Languages, Frameworks and Skills ` are aligned in the center.
- The about section is placed in the center, which aligns with the works section
- The paragraph in the about section takes up 60% width of the parent while the image part takes up the remaining 40% percent

**Please kindly make this adjustment to make it look better** :pray: